vstgui/lib/platform/common/genericoptionmenu.cpp

```diff
@@ -174,6 +174,13 @@ class DataSource : public DataBrowserDelegateAdapter,
 					clickCallback (menu, CDataBrowser::kNoSelection);
 					return 1;
 				}
+				case VKEY_RETURN:
+				case VKEY_ENTER:
+				{
+					if(clickCallback)
```

Comment:
Please use the code style as in the rest of vstgui. We have a space between keyword and open bracket for example. (You can use the .clang-format file which is in the vstgui subdirectory of the repository)